### PR TITLE
docs: modify documentation for CoreChip

### PR DESCRIPTION
**Pull request by anantakumarghosh**
## Description

This PR updates documentation of CoreChip

## Related Issues
#172 

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: app/components/component-docs/dataDisplay/CoreChip.docs.js [hunk 1]
```diff
@@ -199,24 +209,37 @@ Chips with the onDelete prop defined will display a delete icon which changes ap
...
-<CoreStack direction="row" spacing={1}>
-  <CoreChip avatar={<CoreAvatar>M</CoreAvatar>} label="Avatar" />
+        description={
+          <>
+            <CoreTypographyBody1>
+          You can add ornaments to the beginning of the component.
+Use the avatar prop to add an avatar or use the icon prop to add an icon.
+            </CoreTypographyBody1>
+
+            <CoreH6 styleClasses={[]}>Avatar Chip</CoreH6>
```
Comment (techoneel) [inline app/components/component-docs/dataDisplay/CoreChip.docs.js:219]: Empty styleClasses

File: app/components/component-docs/dataDisplay/CoreChip.docs.js [hunk 2]
```diff
@@ -310,48 +343,42 @@ Use the avatar prop to add an avatar or use the icon prop to add an icon."
...
+        code={`<CorePaper styleClasses={[CoreClasses.DISPLAY.FLEX, CoreClasses.ALIGNMENT.JUSTIFY_CONTENT_CENTER, CoreClasses.PADDING.P1]}
   component="ul"
 >
   {[
-    { key: 0, label: 'Angular' },
-    { key: 1, label: 'jQuery' },
-    { key: 2, label: 'Polymer' },
-    { key: 3, label: 'React' },
-    { key: 4, label: 'Vue.js' },
+    { key: 0, label: "Angular" },
```
Comment (techoneel) [inline app/components/component-docs/dataDisplay/CoreChip.docs.js:371]: Change it to Wrappid packages name or something relatable to Wrappid

**Review by techoneel — CHANGES_REQUESTED** [review]
Please change the requested changes.

File: app/components/component-docs/dataDisplay/CoreChip.docs.js (hunk 1 above)
Comment (anantakumarghosh) [reply app/components/component-docs/dataDisplay/CoreChip.docs.js:219]: I suggest adding lint checks for these to the @wrappid/core-team

File: app/components/component-docs/dataDisplay/CoreChip.docs.js (hunk 2 above)
Comment (anantakumarghosh) [reply app/components/component-docs/dataDisplay/CoreChip.docs.js]: Good thought

**Review by techoneel — APPROVED** [review]
Good to go

**Pull request merged**